auto_tuning/proxy/src/proxy-runners/run-all.py

```diff
@@ -0,0 +1,78 @@
+import os, sys, inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
+
+import argparse
+import proxy_bindings as pb
+from progress.bar import Bar
+import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import time
+import os
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-c', '--cells', default=50000, type=int, help="num cells in a time cluster")
+parser.add_argument('-t', '--timesteps', default=30, type=int, help="num time steps/repeats")
+parser.add_argument('--output_type', default='c', type=str, help="console(p), json (j), matplotlib (m)")
+parser.add_argument('-o', '--output_dir', default='.', type=str, help="relative output directory")
+args = parser.parse_args()
+
+if not args.output_type in ['j', 'c', 'p']:
+    print(f'notice: output only to terminal')
+
+kernels = [pb.Kernel.all,
+           pb.Kernel.local,
+           pb.Kernel.neigh,
+           pb.Kernel.ader,
+           pb.Kernel.localwoader,
+           pb.Kernel.neigh_dr,
+           pb.Kernel.godunov_dr]
```

Comment:
We could also use the helper function from Aux that you export. This way, the code would be more resilient.

Reply:
Can you elaborate it? I didn't get it

Reply:
You can export this one function here:
https://github.com/SeisSol/SeisSol/blob/2d9a950251d5ea74303a9613ddc887dacbb0816e/auto_tuning/proxy/src/proxy_common.hpp#L62
And then use this for a list of valid kernels?

Reply:
done